splash/lua_modules/splash.lua

```diff
@@ -223,5 +223,42 @@ function Splash:on_request(cb)
   end)
 end
 
+local Response = {}
+Response.__index = Response
+
+function Response._create(py_reply)
+    local self = {
+        headers=py_reply.headers,
+        url=py_reply.url,
+        status=py_reply.status,
+        info=py_reply.info
+    }
+
+    setmetatable(self, Response)
+
+    for key, value in pairs(py_reply.headers) do 
+        self[key] = value
+    end
```

Comment:
Does it mean that if there is 'url' http header is the response user won't be able to access response.url attribute?